docs/learn/software_overview.md

```diff
@@ -61,27 +62,28 @@ feel free to [edit this document](../index.md#editing-and-contributing).
  - SketchUp
  - OpenSCAD
  - LibreCAD
+ - TinkerCAD
 
 ### Art Design
 
 If you want to carve out art, signs, lettering, or logos then you'll want to use .SVG files.
 These parts have important dimensions and measurements, but they are usually drawn in programs
-specializing in art. The most common programs are inkscape and Adobe Illustrator. In these programs,
+specializing in art. The most common programs are Inkscape and Adobe Illustrator. In these programs,
 you can very precisely adjust the shape to get an aesthetically pleasing design, or import another
 design and adjust it.
 
 !!! note
     If you have a piece of art in a .jpg, .png, or .bmp format, you can't directly use that. To make
-    it usable, you can use inkscape. Try searching for [inkscape trace bitmap
+    it usable, you can use Inkscape. Try searching for [inkscape trace bitmap
     tutorial](https://duckduckgo.com/?q=inkscape+trace+bitmap+tutorial)
 
-## CAM: Computer Assisted Machining
+## Toolpathing-CAM
```

Comment:
I liked the previous title better :)

Reply:
Here is my only important overall suggestion, I think we would be on the same page but let me explain in detail.

My feeling (and essentially the additions I made) are that the steps be described as a workflow as opposed to software items.  Okay, no biggie.  As such each section title should match the title of the workflow step.

So if the workflow step is "CAM" then yes the section title should be "CAM: Computer Assisted Machining". If you guys decide that the workflow step is "Toolpathing" then should call the section title the same.

Reply:
```suggestion
## CAM
```